src/molecules/NavMenuMob.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Box } from 'atomic-layout'
+import styled from 'styled-components'
+import { SubMenu } from './SubMenu'
+
+const StyledUl = styled.ul`
+  padding: 0;
+  margin: 0;
+  list-style: none;
+`
+
+interface Props {
+  data: Menu[]
+}
+
+interface Menu {
```

Comment:
These interfaces `Menu` and `SubMenu` should be removed and you should use the same interfaces imported from `./SubMenu`.